stacks/minstack.js

```diff
@@ -0,0 +1,32 @@
+const Stack = require('./Stack');
+
+function MinStack(stackMaxSize){
+    this.stack = new Stack(stackMaxSize);
+    this.minStack = new Stack(stackMaxSize);
+}
+
+MinStack.prototype.push = function(num) {
+    var currentMinimum = this.minStack.peek();
+    if(this.minStack.count() === 0 || (typeof currentMinimum === "undefined" || currentMinimum > num) )
```

Comment:
this needs more brevity, ideally, number to be pushed in minStack should be decided in if else loop and push in minstack should happen outside it.